### PR TITLE
Process multiple specified docker images correctly

### PR DESCRIPTION
**Pull request by lancelet**
- Fixes Issue #47.

### CODE REVIEW

File: src/test/bash/test-dockermake.sh [hunk 1]
```diff
@@ -116,6 +115,37 @@ chmod +x $TMPPATHORDER/docker
...
+              "--build-arg=http_proxy=fakeproxy" \
+              "--build-arg=https_proxy=fakeproxy" \
+              "--build-arg=ftp_proxy=fakeproxy" \
+              "-t" \
+              "PREFIXtmpDOCKERIMAGExxx:$(cat VERSION)" \
+              "tmpDOCKERIMAGExxx" \
+    )
+
+WVPASSEQ "$(echo ${dockerArgs[@]})" "$(echo ${expected[@]})"
+
```
Comment (lancelet) [inline src/test/bash/test-dockermake.sh]: NOTE: There was an existing test here, which I've just extended.

The original test assumed that `docker` (fake docker in this case) was actually invoked, and tested that the build order of the images was the same as specified on the command line. However, it turns out that `docker` was never actually run, so the test never had the opportunity to check anything.

This modified version simply checks that the fake docker script was run with the correct arguments. It fails without my new modifications to `dockermake.sh` (exercise for the reader to check).

**Comment by lancelet** [comment]
@stephanh I'm not sure what's going on with the coverage stats here. Why is it blocked?

It seems to be complaining that coverage has *decreased* because I added lines to the *tests*:
https://coveralls.io/builds/10389093/source?filename=src%2Ftest%2Fbash%2Ftest-dockermake.sh

(Is that your interpretation of it?)

**Comment by stephanh** [comment]
@lancelet not sure. One problem is that coveralls is also taking our tests into consideration when doing test coverage so when you add extra code to your tests it might flag that as reducing test coverage. @jsok is looking at removing that.

Is your branch rebased on the latest version of ci2?

**Comment by jsok** [comment]
Can you try adding `SimpleCov.add_filter "/src\/test/"` to `.simplecov` . 

See also: [simplecov filtering](https://github.com/colszowka/simplecov#filters)

**Comment by lancelet** [comment]
@jsok I've tried a couple of variations of excluding tests:
```
SimpleCov.add_filter 'src/test/'
```
and
```
SimpleCov.add_filter "/src\/test/"
```

Both (see above) have resulted in test figures around 70%. I'm guessing that the decreased coverage is simply because a greater percentage of code overall was actually being executed when the tests were included.

I'll have a bash at adding a couple of unrelated tests to just push it back above 76% (on the basis that more tests are good regardless 😄). Otherwise it might need manual intervention.

**Comment by lancelet** [comment]
@jsok ok - I've bumped the test coverage up a tiny bit. It's going to take me too long to get it up to its (invalid) former values though.

I think one of the admins of `ci2` will have to reset the acceptable coverage value.

**Comment by jsok** [comment]
Yeah don't beat yourself up about it for vanity reasons. I'm not actually an admin so I can't bypass the status check to merge... @stephanh can you help?

**Comment by stephanh** [comment]
Merged.

**Pull request merged**